numerapi/signalsapi.py

```diff
@@ -264,6 +265,51 @@ def public_user_profile(self, username: str) -> Dict:
         utils.replace(data, "totalStake", utils.parse_float_string)
         return data
 
+    def public_user_profile_v2(self, username: str) -> Dict:
```

Comment:
Does the v1 version provide anything over v2? Similar to my previous comment, I prefer completely replacing the old version.